libs/application/templates/health-insurance/src/forms/messages.ts

```diff
@@ -350,19 +369,19 @@ export const m = defineMessages({
   },
   successfulSubmissionMessage: {
     id: 'hi.application:successfulSubmission.message',
-    defaultMessage:
-      'Númer umsókninnar er **{applicationNumber}**. Tölvupóstur til staðfestingar hefur verið sendur. ',
+    defaultMessage: 'Númer umsókninnar er **{applicationNumber}**.',
     description:
       'Your application number is **{applicationNumber}**. A confirmation e-mail has also been sent. ',
   },
   nextStepReviewTime: {
     id: 'hi.application:nextStep.duration',
     defaultMessage:
-      'Tekið getur allt að 2-6 vikur að meðhöndla umsókn. Eftir því hversu fljótt fyrra tryggingarland þitt svarar fyrirspurn okkar getur það tekið lengur.',
+      'Tekið getur allt að 2-6 vikur að fara yfir umsókn. Lengd svartíma fer eftir því hversu fljótt fyrra tryggingarland þitt svarar fyrirspurn okkar og getur það tekið lengri tíma.',
     description:
-      'An application may take up to 2–6 weeks to process. Depending on how fast your former country of insurance to responds to our request, it could take a longer.',
+      'An application may take up to 2–6 weeks to process. Depending on how fast your former country of insurance is to responds on our request, it may take longer time.',
   },
   nextStepStatusCheck: {
+    // should this be removed?
```

Comment:
If we dont use it anywhere then i think it is better to remove it and we can add it later instead :)